src/settings.json

```diff
@@ -132,5 +133,19 @@
     "1": {"setting_name": "shutter_a", "gpio_pin": 16},
     "2": {"setting_name": "fps", "gpio_pin": 26},
     "3": {"setting_name": "wb", "gpio_pin": 5}
+  },
+  "i2c_oled": {
+    "enabled": true,
+    "width": 128,
+    "height": 64,
+    "font_size": 10,
+    "values": [
```

Comment:
Can be any value read from redis controller plus some more like the cpu/ram stuff.
The order here is the same order it shows on the display, except that resulotion and is_recording always shows on the first row.